read-json.js

```diff
@@ -524,6 +524,33 @@ function final (file, data, log, strict, cb) {
   })
 }
 
+function fillTypes (file, data, cb) {
+  var index = data.main ? data.main : 'index.js'
+  if (data.exports && typeof data.exports === 'string') index = data.exports
+  if (data.exports && data.exports['.']) index = data.exports['.']
+
+  function switchExt (file, ext) {
+    var extless = path.join(path.dirname(file), path.basename(file, path.extname(file)))
+    return './' + extless + '.' + ext
```

Comment:
Is this right? By default things like `main` and `files` don't prepend `./`.  It seems inappropriate here.

Reply:
old things, like main/bin/files/browser, don't require the leading `./`, but new things, namely `exports`, does require it - it seems better to go with that more robust pattern and require it. (not sure if this comment is relevant here or not)